Changelog.md

```diff
@@ -1,5 +1,9 @@
 # Fossa CLI Changelog
 
+## Unreleased
+
+- Makes experimental flags discoverable and documents them. ([#723](https://github.com/fossas/fossa-cli/pull/723))
```

Comment:
Let's do a release since they are becoming discoverable.

Reply:
For sure, this is just a placeholder until I'm about to merge 👍